README.md

```diff
@@ -320,6 +327,57 @@ In order to set these configurations manually, edit `demo_config.h` in `demos/ht
 
 You can generate the presigned urls using [demos/http/common/src/presigned_urls_gen.py](demos/http/common/src/presigned_urls_gen.py). More info can be found [here](demos/http/common/src/README.md).
 
+#### Setup for AWS IoT Jobs demo
+
+1. The demo requires the Linux platform to contain curl and libmosquitto which can be installed with:
+
+```
+    apt install curl libmosquitto-dev
```

Comment:
It is not necessarily required. It will build it from source if not installed.

Reply:
Also not all Linux distros use apt

Reply:
Addressed